packages/sitecore-jss-cli/package.json

```
@@ -12,8 +12,8 @@
     "lint": "eslint ./src/**/*.ts",
     "prepublishOnly": "npm run build",
     "jss": "node ./dist/cjs/bin/jss.js",
-    "test": "mocha --require ts-node/register \"./src/**/*.test.ts\"",
-    "coverage": "nyc npm test"
+    "test": "mocha --require ts-node/register/transpile-only \"./src/**/*.test.ts\"",
+    "coverage": "nyc --require ts-node/register/transpile-only npm test"
```

Is the extra `--require` here necessary?
```suggestion
    "coverage": "nyc npm test"
```

---

yes - otherwise coverage command fails